src/components/RepoCard.tsx

```diff
@@ -0,0 +1,64 @@
+import '@styles/RepoCard.scss'
+import { RoundedTag } from './RoundedTag'
+import { Col, Row, Typography } from 'antd'
+import { StarOutlined } from '@ant-design/icons'
+import '@styles/RepoCard.scss'
+import { GithubRepoContent } from '@types'
+
+const { Title, Text } = Typography
+
+interface RepoCardProp {
+  repo: GithubRepoContent
+}
+export const RepoCard = ({ repo }: RepoCardProp) => {
+  interface EmojiMap {
+    [key: string]: string
+  }
+  const emojiMap: EmojiMap = {
+    ':eyes:': '👀',
+    ':pencil2:': '✏️',
+  }
+
+  const convertShortcodesToUnicode = (text: string) => {
+    const regex = /(:\w+:)/g
+    return text.replace(regex, (match) => emojiMap[match] || match)
+  }
```

Comment:
this function also doesn't necessarily need to be defined as a component method, can be defined outside of it